src/table/resizer/index.tsx

```diff
@@ -169,20 +181,51 @@ export function Resizer({
           setIsDragging(true);
         }}
         onClick={() => {
-          // Prevents dragging mode activation for VO+Space click.
+          // Prevent mouse drag activation and activate keyboard dragging for VO+Space click.
           setIsDragging(false);
+          setResizerHasFocus(true);
+          setIsKeyboardDragging(true);
+          resizerSeparatorRef.current?.focus();
         }}
         onFocus={() => {
-          setHeaderCellWidth(getHeaderWidth(resizerRef.current));
+          setHeaderCellWidth(getHeaderWidth(resizerToggleRef.current));
           setResizerHasFocus(true);
         }}
-        onBlur={() => {
-          setResizerHasFocus(false);
+        onBlur={event => {
+          if (event.relatedTarget !== resizerSeparatorRef.current) {
+            setResizerHasFocus(false);
+          }
         }}
-        {...resizerAriaProps}
+        // Using a custom role-description to make the element's purpose better clear.
+        // The role-description must include the word "button" to imply the interaction model.
+        // See https://www.w3.org/TR/wai-aria-1.1/#aria-roledescription
+        aria-roledescription={roleDescription}
+        aria-labelledby={ariaLabelledby}
         tabIndex={tabIndex}
         data-focus-id={focusId}
       />
+      <span
+        ref={resizerSeparatorRef}
+        id={separatorId}
+        role="separator"
+        tabIndex={-1}
+        aria-hidden={!isKeyboardDragging}
+        aria-orientation="vertical"
+        aria-valuenow={headerCellWidth}
+        // aria-valuetext is needed because the VO announces "collapsed" when only aria-valuenow set without aria-valuemax
+        aria-valuetext={headerCellWidth.toFixed(0)}
+        aria-valuemin={minWidth}
+        data-focus-id={focusId}
+        onBlur={event => {
+          setResizerHasFocus(false);
+          if (isKeyboardDragging) {
+            setIsKeyboardDragging(false);
+          }
+          if (event.relatedTarget !== resizerToggleRef.current) {
```

Comment:
What does this check mean? Might be worth a comment.

Reply:
Here, we have two actual elements that juggle focus -- the activation button and the separator. When there is a blur event we want to commit an update but not when the focus juggles between these two elements because that is merely an implementation detail.

Reply:
I can add a code comment if you think that makes sense.

Reply:
Yes, correct. I will add a code comment

Reply:
Actually, I will add a code comment to another similar line is the file and remove the IF statement from here - it is not necessary to prevent the blur when focus moves back to the resizer button!